lib/cocoapods/target.rb

```diff
@@ -158,12 +158,6 @@ def module_map_path
       support_files_dir + "#{label}.modulemap"
     end
 
-    # @return [Pathname] the absolute path of the prefix header file.
```

Comment:
Method moved to `pod_target.rb`. Noone except `pod_target_installer.rb` is invoking this method. It's pointless to have it on the base class.